server/events/terraform/filter/filter.go

```diff
@@ -0,0 +1,18 @@
+package filter
+
+import (
+	"regexp"
+)
+
+type LogFilter struct {
+	Regexes []*regexp.Regexp
+}
+
+func (l *LogFilter) LogLineShouldBeFiltered(message string) bool {
```

Comment:
```suggestion
func (l *LogFilter) ShouldFilterLine(message string) bool {
```
seems a bit unnecessarily verbose, log is already in the struct name.